guide/principles/principles.md

```diff
@@ -0,0 +1,164 @@
+
+And while Bitcoin is already widely used by a large number of people, it pales in comparison with the billions that will use it in the future. Without dumbing down or reducing the possibilities we need to design products that are prepared for people that are new to Bitcoin. This means using plain and ordinary language, explaining things in the context where they are needed, not overwhelming people with technical detail, and more.
+
+*Provide equal and direct access to the Bitcoin network, without middlemen and exclusive requirements.* <br>
+*Design Bitcoin products that are usable by the widest range of people possible.*
+
+**Do**
+- Use plain language that people new to Bitcoin can understand regardless of prior knowledge
+- Use more languages than just english
+- Educate in place, when people are presented with a new concept
+- Treat users with impairments, or who rely on assistive technologies, as first-class citizens
+
+**Don't**
+- Introduce requirements that might exclude many people, for example by building only for high-end hardware, requiring US KYC (Know-your-customer) information etc
+- Avoid technical detail that is not required knowledge, or technical terms like seed phrase XPUBs, mnemonics etc.
+- Do not put all education up front and expect people to read and remember it
+
+---
+
+## Transparency
+
+While an open and decentralized financial system that lets people connect directly with it is great, it does put a lot of burden on the users to pick a product they trust, and that works for them. We can make this easier by freely sharing information about how our products work, what technologies they use and rely on and so forth. Let people verify that your claims are true. It is also very important to be open about what risks the user might be taking on with self-custody, under which scenarios they might lose access to their funds and best practices for avoiding this. 
+
+*Be open and transparent with how your product works.* <br>
+*Explain what risks the user is taking on, and how best to mitigate them.*
+
+**Do**
+- Build your product as open source when possible
+- Let people verify, without having to trust
+- Be clear about what risk the user is taking on, and how they can lose access to their funds 
+
+**Don't**
+- Make claims that are not explained or verifiable 
+
+---
+
+## Self-custody
+
+Our existing mental models of access to digital services are usernames and passwords, controlled by a company that has custody of your funds and data. With everyone having direct access to the Bitcoin network without middlemen, there is no longer a need for people to delegate control of their funds. While it comes with greater responsibility, it is self-custody that really enables the open financial system of peer-to-peer transactions.
+
+*Give end users full control of their funds.*
+
+**Do**
+- Let users control their private keys, and funds directly
+
+**Don't**
+- Custody funds for your users
+
+---
+
+## Security
```

Comment:
You think it's worth noting that security is also a feeling? My wallet can have the best private key management but I might still not feel secure when it looks crappy, or there's something not clear about the backup mechanisms.

Reply:
Added. 7b8e2f3